src/components/TreeView/ControlledTreeView.tsx

```diff
@@ -7,6 +7,7 @@ import Loading from '~/icons/Loading';
 import ChevronDown from '~/icons/ChevronDown';
 import ChevronRight from '~/icons/ChevronRight';
 import {Typography} from '~/components/Typography';
+import {isDisabled} from "@testing-library/user-event/dist/utils";
```

Comment:
Why do you import something from `testing-library` into the component?

Reply:
no idea :-)